### PR TITLE
Merging styling changes

### PR DESCRIPTION
**Pull request by haydennharper**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -18,55 +18,150 @@ const authenticateSchema = new mongoose.Schema({
...
+  },
+  authors: {
+    type: Array,
+    unique: false
+  },
+=======
   gBookId: { type: String, unique: false}, //googleapi book ID
   title: { type: String, unique: false },
   authors: { type: Array, unique: false },
+>>>>>>> bd34a6f0950da10061ffd0fd6f366c7243ea0659
```
Comment (ks10825n) [inline database/index.js:38]: I think these arrows will cause conflicts

File: database/index.js [hunk 2]
```diff
@@ -18,55 +18,150 @@ const authenticateSchema = new mongoose.Schema({
...
+  },
   userBooks: [userBookSchema],
+<<<<<<< HEAD
+  friends: [{
+    _id: false,
+    username: String
+  }],
+=======
   friends: Array,
+>>>>>>> bd34a6f0950da10061ffd0fd6f366c7243ea0659
```
Comment (ks10825n) [inline database/index.js:95]: I think these arrows will cause conflicts

File: database/index.js [hunk 3]
```diff
@@ -18,55 +18,150 @@ const authenticateSchema = new mongoose.Schema({
...
+      type: Number
+    },
+=======
     username: {type: String},
     review_date: {type: Date, default: Date.now},
     rating: {type: Number, min: 1, max: 5},
     review_body: {type: String, min: 25, max: 1000},
     reported_review: {type: Boolean},
     helpful_review: {type: Number},
+>>>>>>> bd34a6f0950da10061ffd0fd6f366c7243ea0659
```
Comment (ks10825n) [inline database/index.js:143]: I think these arrows will cause conflicts

File: database/models/bookdata.js [hunk 4]
```diff
@@ -1,16 +1,35 @@
...
 
 const getAllBookData = async (book_id) => {
   const api_data = await axios.get(`https://www.googleapis.com/books/v1/volumes?q=${book_id}`);
+<<<<<<< HEAD
+  let metadata = await BookData.find({
+    lookup_id: book_id
+  });
+=======
   let metadata = await BookData.find({ lookup_id: book_id });
+>>>>>>> bd34a6f0950da10061ffd0fd6f366c7243ea0659
```
Comment (ks10825n) [inline database/models/bookdata.js:13]: I think these arrows will cause conflicts

File: database/models/bookdata.js [hunk 5]
```diff
@@ -36,20 +61,53 @@ const addBookComment = async (book_id, review_id, comment) => {
...
+    lookup_id: book_id,
+    "reviews.review_id": review_id
+  }, {
+    $push: {
+      "reviews.$.comments": newComment
+    }
+  });
+=======
   const result = await BookData.updateOne({ lookup_id: book_id,  "reviews.review_id": review_id }, { $push: { "reviews.$.comments": newComment }});
+>>>>>>> bd34a6f0950da10061ffd0fd6f366c7243ea0659
```
Comment (ks10825n) [inline database/models/bookdata.js:75]: I think these arrows will cause conflicts

File: database/models/bookdata.js [hunk 6]
```diff
@@ -67,4 +125,5 @@ const markReviewComment = async (book_id, review_id, comment_id, mark_type) => {
     );
     return result;
   }
-}
\ No newline at end of file
+}
+>>>>>>> bd34a6f0950da10061ffd0fd6f366c7243ea0659
```
Comment (ks10825n) [inline database/models/bookdata.js:129]: I think these arrows will cause conflicts

**Review by ks10825n — COMMENTED** [review]
Hey I think I see some remnants of a merge conflict in the database files. Might be easier to just delete these specific ones and pull them down again since you won't have any changes on your end.

Or you can clear as best you can and accept income changes if there's more merge conflicts coming from back-end

**Comment by anerolabs** [comment]
Closed and opened new PR without the conflicted DB files.

**Pull request closed without merging**